ovn-tester/ovn_utils.py

```diff
@@ -37,13 +87,22 @@ def del_port(self, port):
 
     def bind_vm_port(self, lport):
         cmd = f'bash -c \'ip netns add {lport.name} ; ' \
+              f'ip netns exec {lport.name} sysctl -w ' \
+              f'net.ipv6.conf.all.optimistic_dad=1 ; ' \
               f'ip link set {lport.name} netns {lport.name} ; ' \
               f'ip -n {lport.name} link set {lport.name} ' \
-              f'address {lport.mac} ; ' \
-              f'ip -n {lport.name} addr add {lport.ip}/{lport.plen} ' \
-              f'dev {lport.name} ; ' \
-              f'ip -n {lport.name} link set {lport.name} up ; ' \
-              f'ip -n {lport.name} route add default via {lport.gw}\''
+              f'address {lport.mac} ;'
+        if lport.ip:
+            cmd += f' ip -n {lport.name} addr add {lport.ip}/{lport.plen} ' \
+                   f'dev {lport.name} ; ' \
+                   f'ip -n {lport.name} link set {lport.name} up ; ' \
```

Comment:
Since the v6 path is missing the up it would be better to extract it outside of the if.

Reply:
@almusil ack, I will fix it, thx